cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/Outlet.java

```diff
@@ -0,0 +1,124 @@
+package mil.army.usace.hec.cwms.radar.client.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Objects;
+
+
+/**
+ * Outlet
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+@jakarta.annotation.Generated(value = "io.swagger.codegen.v3.generators.java.SpringCodegen",
+        date = "2024-07-29T13:57:29.992917800-07:00[America/Los_Angeles]")
+public class Outlet {
+
+    @JsonProperty("project-id")
+    private CwmsId projectId = null;
+
+    @JsonProperty("location")
+    private Location location = null;
+
+    @JsonProperty("rating-group-id")
+    private CwmsId ratingGroupId = null;
+
+    @JsonProperty("rating-category-id")
+    private CwmsId ratingCategoryId = null;
```

Comment:
@adamkorynta, this might be problematic, since the ratingCategoryId in the CDA code is effectively read-only.  Do you have any recommendations on how to handle that in the code gen class?

Reply:
That's a good point. I don't know what the codegen does to mark such things. You can however annotate that in the CDA DTO's though we haven't done it  yet: https://swagger.io/docs/specification/data-models/data-types/

Reply:
I'll take a look at this in a bit and see what I can do

Reply:
Made a ticket #205 to address this.  @adamkorynta please resolve the remaining conversations, and I'll approve the change.